src/metabase/integrations/ldap.clj

```diff
@@ -177,16 +177,21 @@
      :filter     (str/replace (ldap-user-filter) filter-placeholder (Filter/encodeValue username))
      :size-limit 1})))
 
+(defn- lower-case-attribute-names
```

Comment:
I've noticed that what this function does is actually much more general than what's applicable to LDAP attributes: it lower cases a map's keys. As such, it's probably better to move it to the `metabase.util` namespace, and rename it to suit it's more general purpose. I suggest `lower-map-keys`, similar to `metabase.util/recursive-map-keys`.

Please note that the idiomatic name for a general map, is `m`. So please rename `record` to `m` while you're at it.

Remember to update the doc string. :slightly_smiling_face:

Reply:
This should also be done.